### ISSUE TITLE
Task entry that breaks taskwarrior

### ISSUE THREAD

**Issue by hwine** [GothenburgBitFactory/taskwarrior#2128]
I isolated the entry below as being the cause of the problem (orignailly reported [here](https://github.com/GothenburgBitFactory/taskwarrior/issues/2044#issuecomment-469857187)). At some point, I did get heavy handed with using the ``wait:later`` modification, which I assume is how the record got that way. I didn't notice any issue immediately, and *think* the problem output only started happening when the normal "generate next recurrence" happened.

* What command(s) did you run?
```bash
mkdir /tmp/confuse_tw
cd /tmp/confuse_tw
export TASKRC=$PWD/.taskrc 
export TASKDATA=$PWD/
touch $TASKRC
cat >pending.data <<EOF
[description:"Report on last month's fx-sig-verify work" due:"1522566000" entry:"1522365924" mask:"+-+++++-----" modified:"1549660568" project:"work" recur:"monthly" reviewed:"1545369114" status:"recurring" tags:"status" uuid:"e54c3641-b1af-4ac9-8216-ed6c1149572f" wait:"2147414400"]
EOF
task
```
* What did you expect to happen?
normal output
* What actually happened?
*N.B. the integer before my prompt is the exit code of the prior command*
```bash
$ mkdir /tmp/confuse_tw
 0 [hwine@hwine-surface] /tmp
$ cd /tmp/confuse_tw
 0 [hwine@hwine-surface] /tmp/confuse_tw
$ export TASKRC=$PWD/.taskrc
 0 [hwine@hwine-surface] /tmp/confuse_tw
$ export TASKDATA=$PWD/
 0 [hwine@hwine-surface] /tmp/confuse_tw
$ touch $TASKRC
 0 [hwine@hwine-surface] /tmp/confuse_tw
$ cat >pending.data <<EOF
> [description:"Report on last month's fx-sig-verify work" due:"1522566000" entry:"1522365924" mask:"+-+++++-----" modified:"1549660568" project:"work" recur:"monthly" reviewed:"1545369114" status:"recurring" tags:"status" uuid:"e54c3641-b1af-4ac9-8216-ed6c1149572f" wait:"2147414400"]
> EOF
 0 [hwine@hwine-surface] /tmp/confuse_tw
$ task
TASKRC override: /tmp/confuse_tw/.taskrc
TASKDATA override: /tmp/confuse_tw/
[task next]
Warning: You have specified that the 'wait' date is after the 'due' date.
'2178950400' is not a valid date in the '' format.
 
```
* Paste the output of the ```task diag``` command.
```
$ task diagnostics
TASKRC override: /tmp/confuse_tw/.taskrc
TASKDATA override: /tmp/confuse_tw/

task 2.5.1
   Platform: Linux

Compiler
    Version: 7.2.0
       Caps: +stdc +stdc_hosted +LP64 +c8 +i32 +l64 +vp64 +time_t64
 Compliance: C++11

Build Features
      CMake: 3.9.5
    libuuid: libuuid + uuid_unparse_lower
  libgnutls: 3.5.8
 Build type: None

Configuration
       File: /tmp/confuse_tw/.taskrc (found), 0 bytes, mode 100644
       Data: /tmp/confuse_tw/ (found), dir, mode 40755
     TASKRC: /tmp/confuse_tw/.taskrc
   TASKDATA: /tmp/confuse_tw/
    Locking: Enabled
         GC: Enabled
    $VISUAL: nvim
     Server:
      Trust: strict
Certificate: , not readable, 0 bytes
        Key: , not readable, 0 bytes
    Ciphers: NORMAL
      Creds:

Hooks
     System: Enabled
   Location: /tmp/confuse_tw//hooks
             (-none-)

Tests
      $TERM: tmux-256color (177x112)
       Dups: Scanned 1 tasks for duplicate UUIDs:
             No duplicates found
 Broken ref: Scanned 1 tasks for broken references:
             No broken references found

 0 [hwine@hwine-surface] /tmp/confuse_tw
$ wc -l .??* *
  0 .taskrc
  0 completed.data
  1 pending.data
  1 total
 0 [hwine@hwine-surface] /tmp/confuse_tw
$
```

**Comment by zarelit** [comment]
@tbabej  Can this be related to  #2131 ? The description starts with "report"

**Comment by hwine** [comment]
@zarelit @tbabej -- unfortunately changing the text is not a workaround:
```bash
$ cat >pending.data <<EOF
> [description:"XXX on last month's fx-sig-verify work" due:"1522566000" entry:"1522365924" mask:"+-+++++-----" modified:"1549660568" project:"work" recur:"monthly" reviewed:"1545369114" status:"recurring" tags:"status" uuid:"e54c3641-b1af-4ac9-8216-ed6c1149572f" wait:"2147414400"]
> EOF
 0 [hwine@hwine-surface] /tmp/confuse_tw
$ task
TASKRC override: /tmp/confuse_tw/.taskrc
TASKDATA override: /tmp/confuse_tw/
[task next]
Warning: You have specified that the 'wait' date is after the 'due' date.
'2178950400' is not a valid date in the '' format.
```

**Comment by tbabej** [comment]
Thank you for reporting @hwine.

This seems to be a manifestation of the Y2038 problem (see #2373), which is fixed on 2.6.0 which supports 64-bit timestamps:

![image](https://user-images.githubusercontent.com/2123868/105646726-7b6c2d80-5e6f-11eb-8647-563fc119a35c.png)

(note the wait attribute value)

Reproduced on 2.5.1. Fixed on 2.6.0 via 9ed047d46de4007b875da9c2a373e0da1be63fd7 and 6bbfcba738c568e163ee0a38c1e33005e42fc125.
